/graphslam_prefiltering_node:
    ros__parameters:
        leaf_size: 0.05
        random_sample_num: 2000.0
        mean_k: 30
        stddev: 1.2
        min_x: -100.0
        max_x: 100.0
        min_y: -100.0
        max_y: 100.0
        min_z: -100.0
        max_z: 100.0
        min_distance_cloud: 0.1
        max_distance_cloud: 120.0

        lidar_topic: /velodyne_points

